src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --nav-color: #00acdc;
+}
+
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  line-height: 36px;
+  text-transform: uppercase;
+}
+
+.nav-item {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: center;
+  padding-left: 0;
+}
+
+.nav-link {
```

Comment:
it seems like now font doesn't match the design:

![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/24a253e2-4f96-4ea2-82ef-7557b70e6b55)